react.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     'react/prop-types': 'off',
     'react/jsx-filename-extension': 'off',
     'react/require-default-props': 'off',
-    'react/no-unknown-property': ['error', { ignore: ['css', 'columns'] }],
+    'react/no-unknown-property': 'off', // Allows custom properties to be used for css (is-desktop, etc) and for parents (columns, growX, etc)
```

Comment:
Note from Jordi:
turns off react/no-unknown-property -- Allows custom properties to be used for css (is-desktop, etc) and for parents (columns, growX, etc)

before we couldn't do this:
![image](https://user-images.githubusercontent.com/3885959/215877571-28d805f4-31d0-4365-b152-74f69c0dff31.png)

We could always put our custom css props under data-* and add the parent props like growX as exceptions to the rule. FYI, TypeScript, doesn't have the data- requirement, it only needs a hyphen in the propname for it to be ok with the custom prop. And it checks unknown props using TS, not eslint. And we add props for parents in a .d.ts file that updates Attributes in react (see zion's globals-react.d.ts file)

![image](https://user-images.githubusercontent.com/3885959/215877683-fd85540c-d100-40d8-85a8-fc393ad74b07.png)

In a TS file:

![image](https://user-images.githubusercontent.com/3885959/215877719-26cf43d3-ae3c-4dec-82f2-b110d6d8f458.png)

![image](https://user-images.githubusercontent.com/3885959/215877798-0c1ed7ca-8bfb-45a3-8b3a-26e31065c29b.png)
![image](https://user-images.githubusercontent.com/3885959/215877816-4990d2b6-0d6b-4c2a-87a4-09db1d2c13fc.png)

So, I could see not wanting to turn this off for the org, but then we need to update it to allow growX and such.

Reply:
Does taking the `css` out of the ignore array break people's apps who use Emotion?

Reply:
No, the entire rule was turned off, so it won't introduce lint, it would just remove it.